app/controllers/concerns/idv/verify_info_concern.rb

```diff
@@ -34,17 +34,17 @@ def shared_update
         user_id: current_user.id,
         threatmetrix_session_id: idv_session.threatmetrix_session_id,
         request_ip: request.remote_ip,
-        double_address_verification: capture_secondary_id_enabled,
+        double_address_verification: double_address_verification,
       )
 
       return true
     end
 
     private
 
-    def capture_secondary_id_enabled
-      current_user.establishing_in_person_enrollment&.
-          capture_secondary_id_enabled || false
+    def double_address_verification
+      # if in person return true else return false
+      analytics_arguments[:analytics_id] == 'In Person Proofing'
```

Comment:
The value checked here is specifically for analytics, so it's safer to check the enrollment directly.

```suggestion
      current_user.has_in_person_enrollment?
```